.circleci/config.yml

```diff
@@ -1,15 +1,21 @@
 # WARNING: DO NOT EDIT THIS FILE DIRECTLY!!!
 # See the README.md in this directory.
 
-# IMPORTANT: To update Docker image version, please first update
-# https://github.com/pytorch/ossci-job-dsl/blob/master/src/main/groovy/ossci/pytorch/DockerVersion.groovy and
```

Comment:
Do we still need to modify this file? This file seems to be missing the docker version `a8006f9a-272d-4478-b137-d121c6f05c83` we are currently using.

Reply:
@ezyang I just updated this PR to remove groovy related text.